modules/nf-core/svync/tests/main.nf.test

```diff
@@ -46,8 +46,29 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
-                    process.out.vcf[0][0],
-                    path(process.out.vcf[0][1]).linesGzip[20..30]
+                    process.out.collectEntries { key, value ->
+                        if(key.matches("^\\d+\$")) {
```

Comment:
What does this condition check here?

Reply:
It avoids having double entries for each process output. It's a bug in nf-test that I prevent this way. This if statement basically checks if the output key is an integer and removes it accordingly

Reply:
I see!